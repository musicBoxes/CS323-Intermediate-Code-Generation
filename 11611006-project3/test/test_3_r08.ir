FUNCTION main :
READ n
IF n > #0 GOTO label_0
GOTO label_1
LABEL label_0 :
WRITE #1
GOTO label_4
LABEL label_1 :
IF n < #0 GOTO label_2
GOTO label_3
LABEL label_2 :
WRITE #-1
GOTO label_4
LABEL label_3 :
WRITE #0
LABEL label_4 :
RETURN #0
