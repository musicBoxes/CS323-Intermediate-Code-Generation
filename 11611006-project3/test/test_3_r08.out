FUNCTION main :
READ dyj
n := dyj
IF n > #0 GOTO label0
GOTO label1
LABEL label0 :
GOTO label5
LABEL label1 :
IF n < #0 GOTO label2
GOTO label3
LABEL label2 :
GOTO label4
LABEL label3 :
LABEL label4 :
LABEL label5 :
RETURN #0
